packages/translated-components/src/index.js

```diff
@@ -41,18 +27,27 @@ const translated = ({
 }) => {
   const warmTranslations = preHeat(translations, format)
```

Comment:
WTH is a warm translation? 😁 🔥

Reply:
#performance

Reply:
Honestly? I can't be sure what Ben meant 😛 .

At a *guess*, I'd say he means pre-processing the translations to interpolate their parameters and do any money-localisation, etc. Purely based on what I see the code doing. Why that's 'pre-warming' is anyone's guess.

Reply:
Is it to warm up the...
` import kebabCase from 'lodash/kebabCase'`

Reply:
Well I mean, nobody likes cold kebab meat.

We should just rename this library to HSP.